scripts/validation/validate-examples.js

```diff
@@ -0,0 +1,35 @@
+const { execSync } = require('child_process');
+const glob = require('glob');
+
+// Use glob to find all YAML files in the examples directory
+const files = glob.sync('./examples/**/*.{yml,yaml}');
+
+let filesCount = 0;
+let errorFilesCount = 0;
+let filesWithErrors = []; // Array to store files that failed validation
+// Validate each file using AsyncAPI CLI
+files.forEach((file) => {
+  filesCount++;
+  try {
+    console.log(`\nValidating: ${file}`);
+    execSync(`npx asyncapi validate ${file}`, { stdio: 'inherit' });
+  } catch (error) {
+    console.error(`Validation failed for: ${file}\n`);
+    errorFilesCount++;
+    filesWithErrors.push(file);
+    // process.exit(1);
```

Comment:
This should be removed. Instead, the process should exit with code 1 after printing all validation errors in https://github.com/asyncapi/spec/pull/1046/files?diff=unified&w=0#diff-db990a3ee807308932214a7f5f1e6e3c279959f27c1c5449d0dfce6e440edcddR32

Reply:
resolved with abbda0d